bitnami/nginx/templates/hpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: {{ include "common.capabilities.hpa.apiVersion" . }}
```

Comment:
I think that we may have a conflict with this change and the changes on lines 26-27. I put a [comment on the commit](https://github.com/bitnami/charts/commit/624e1bffc0d29b45d5d09c56ef35b4ea1e485f62#r73791839) about it.

Reply:
It was fixed, thanks!

Reply:
Thanks for the heads up! We took notice of this issue and addressed it in a later PR, https://github.com/bitnami/charts/pull/10332

Reply:
I tried it out in chart version 11.1.5 and it works now with a server version less than 1.23.0. Thanks!

(The issue was originally seen in version 11.1.2)